cmake/install/Findo3de.cmake.in

```diff
@@ -12,7 +12,15 @@ include(FindPackageHandleStandardArgs)
 
 # This will be called from within the installed engine's CMakeLists.txt
 macro(ly_find_o3de_packages)
-@FIND_PACKAGES_PLACEHOLDER@
+    if(LY_MONOLITHIC_GAME)
+        set(monolithic_file "${LY_ROOT_FOLDER}/Platform/${PAL_PLATFORM_NAME}/Monolithic/o3de_subdirectories_${PAL_PLATFORM_NAME_LOWERCASE}.cmake")
+        if(NOT EXISTS ${monolithic_file})
+            message(FATAL_ERROR "O3DE SDK was not generated to support monolithic builds")
+        endif()
+        include("${monolithic_file}")
+    else()
+        include("cmake/Platform/${PAL_PLATFORM_NAME}/Default/o3de_subdirectories_${PAL_PLATFORM_NAME_LOWERCASE}.cmake")
```

Comment:
Why are these paths different?

Shouldn't the monolithic include be
"**cmake**/Platform/${PAL_PLATFORM_NAME}/Monolithic/o3de_subdirectories_${PAL_PLATFORM_NAME_LOWERCASE}.cmake" with "cmake" instead of ${LY_ROOT_FOLDER}?

Reply:
yeap, my mistake... I also just found that install wont stomp over files that are already there, so I will start cleaning every time I run